map/util/lanelet2_map_preprocessor/CMakeLists.txt

```diff
@@ -1,56 +1,43 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(lanelet2_map_preprocessor)
 
-add_compile_options(-std=c++14)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Werror)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  lanelet2_extension
-  pcl_ros
-  roscpp
-)
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
-catkin_package(
-)
+ament_auto_add_executable(fix_z_value_by_pcd src/fix_z_value_by_pcd.cpp)
 
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
-)
+ament_auto_add_executable(transform_maps src/transform_maps.cpp)
+
+# add_executable(merge_close_lines src/merge_close_lines.cpp)
+# add_dependencies(merge_close_lines ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+# target_link_libraries(merge_close_lines ${catkin_LIBRARIES})
+
+# add_executable(merge_close_points src/merge_close_points.cpp)
+# add_dependencies(merge_close_points ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+# target_link_libraries(merge_close_points ${catkin_LIBRARIES})
+
+# add_executable(remove_unreferenced_geometry src/remove_unreferenced_geometry.cpp)
+# add_dependencies(remove_unreferenced_geometry ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+# target_link_libraries(remove_unreferenced_geometry ${catkin_LIBRARIES})
+
+# add_executable(fix_lane_change_tags src/fix_lane_change_tags.cpp)
+# add_dependencies(fix_lane_change_tags ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+# target_link_libraries(fix_lane_change_tags ${catkin_LIBRARIES})
```

Comment:
Does it mean that some nodes have been ported and some still need to be done?

Reply:
Resolved by #847.